server/api/batch_inventory.py

```diff
@@ -11,7 +11,8 @@
 import io
 from typing import TypedDict
 import uuid
-from xml.etree import ElementTree
+from defusedxml.ElementTree import parse as parse_xml
+from xml.etree.ElementTree import ElementTree
```

Comment:
I still need to import from `xml` since `defusedxml` does not seem to export the `xml.etree.ElementTree.ElementTree` and `xml.etree.ElementTree.Element` values. I don't love that because it means we need to be pretty careful to not accidentally reintroduce an `ElementTree.parse` call somewhere.